float32[] boxes
uint32[] labels
